shared/version/api.go

```diff
@@ -382,6 +382,7 @@ var APIExtensions = []string{
 	"zfs_delegate",
 	"operations_get_query_all_projects",
 	"metadata_configuration",
+	"syslog_socket",
```

Comment:
The code expects OVN logs to be received on that socket so should we include `ovn` in the config key and API key names?

Reply:
Currently it only supports OVN logs. But it could extended in future. So I think the extension name is appropriate. The same goes for the config key.